docs/how-to-profile-a-local-unity-build-remotely.md

```diff
@@ -25,3 +24,5 @@ codeUrl: resolveWithBaseUrl('unity.wasm')
 - If the data is not transmitted automatically, fiddle with toggling/untoggling the record button in the Profiler until the samples start arriving at Unity
 
 - Recommended setup: Only CPU, Rendering and Memory modules on display; Disable VSync and "Others" from CPU module.
+
+- On WebGL the process of building via the editor `bakes` the `local ip` into the build itself. If this changes, it will no longer work and you will have to build again.
```

Comment:
```suggestion
- On WebGL the process of building via the editor bakes-in the `local ip` into the build itself. This means that the profiling build will strictly only work if the machine that compiled it has the same IP. If your IP changes, profiling will no longer work and you will have to build again.
```

Reply:
I would also specify that the "ip baking" only occurs when building with `Autoconnect to profiler` is ticked.